sites/fast-creator/app/creator.tsx

```diff
@@ -111,254 +87,199 @@ class Creator extends Foundation<CreatorHandledProps, {}, CreatorState> {
+                                onUpdateWidth={this.onUpdateWidth}
+                                responsive={true}
+                            />
                         </div>
-                        <Pane>
-                            <ModularForm
-                                messageSystem={fastMessageSystem}
-                                controls={[
-                                    new StandardControlPlugin({
-                                        type: ControlType.linkedData,
-                                        context: ControlContext.fill,
-                                        control: (
-                                            config: ControlConfig
-                                        ): React.ReactNode => {
-                                            return (
-                                                <LinkedDataControl
-                                                    {...config}
-                                                    onChange={this.handleAddLinkedData(
-                                                        config.onChange
-                                                    )}
-                                                />
-                                            );
-                                        },
-                                    }),
-                                ]}
+                        <Background value={neutralLayerL1} className={"dev-tools"}>
+                            <div
+                                ref={this.editorContainerRef}
+                                style={{ height: "100%", paddingTop: "24px" }}
+                            />
+                            <ActionToggle
+                                appearance={ActionToggleAppearance.stealth}
+                                selectedLabel={"Development tools expanded"}
+                                selectedGlyph={downChevron}
+                                unselectedLabel={"Development tools collapsed"}
+                                unselectedGlyph={upChevron}
+                                selected={this.state.devToolsVisible}
+                                onToggle={this.handleDevToolsToggle}
+                                style={{
+                                    position: "absolute",
+                                    top: "12px",
+                                    right: 0,
+                                }}
                             />
-                        </Pane>
-                    </Row>
-                </Container>
+                        </Background>
+                    </div>
+                </div>
+                <div className={this.paneEndClassNames}>
+                    <ModularForm
+                        messageSystem={this.fastMessageSystem}
+                        controls={[
+                            new StandardControlPlugin({
+                                type: ControlType.linkedData,
```

Comment:
Should this instance be cached so you don't have to new up a `StandardControlPlugin` every render?